public/app/features/dashboard/utils/dashboard.ts

```diff
@@ -20,6 +20,19 @@ export function onCreateNewPanel(dashboard: DashboardModel, datasource?: string)
   return newPanel.id;
 }
 
+export function onCreateNewWidgetPanel(dashboard: DashboardModel, widgetType: string): number | undefined {
+  const newPanel: Partial<PanelModel> = {
+    type: widgetType,
+    title: 'Panel Title',
```

Comment:
Maybe we can refactor this into 
```suggestion
    title: 'Widget title',
```
Since we want to move into Viz and widget without talking about the panel.

Reply:
sounds good. thoughts? @axelavargas

Reply:
It also makes sense for me 😄